crypto/mbedtls/Kconfig

```diff
@@ -109,6 +109,15 @@ config MBEDTLS_SSL_DTLS_BADMAC_LIMIT
 	bool "Enable support for a limit of records with bad MAC."
 	default n
 
+config MBEDTLS_SSL_DTLS_CONNECTION_ID
+	bool "Enable the Connection ID extension."
+	default n
+
+config MBEDTLS_SSL_DTLS_CONNECTION_ID_COMPAT
```

Comment:
do you need change mbedtls_config.h

Reply:
#ifdef CONFIG_MBEDTLS_SSL_DTLS_CONNECTION_ID
#define MBEDTLS_SSL_DTLS_CONNECTION_ID
#endif

#ifdef CONFIG_MBEDTLS_SSL_DTLS_CONNECTION_ID_COMPAT
#define MBEDTLS_SSL_DTLS_CONNECTION_ID_COMPAT  0
#endif
These two have already be controlled by kconfig